client/src/components/CreateMeetup/CreateMeetup.js

```diff
@@ -68,6 +85,44 @@ function CreateMeetup() {
     "cooking",
   ];
 
+  useEffect(async () => {
+    // TODO: redirect to login page for unauthenticated users.
+    // - Redux store may need a 'logging in' action to distinguish between unauthenticated
```

Comment:
out of curiosity, when would we have a situation when an authenticated user's id wasn't loaded in the store yet?

Reply:
If you log in, close all DownToMeet tabs, then open a fresh tab directly to `/meetup/:id/edit` or `/create`, userID will need to be repopulated from the `useEffect()` in `App.js`.

Reply:
To elaborate, redirecting to `/login` on mount if `userID` is not found doesn't work due to this. If you open `/create` on a fresh tab you'll be redirected to `/login` even if you're already authenticated. You can't just recheck `userID` when it's loaded into the store either, since you'll already have been redirected to a different page. 

If we want the page to redirect to `/login` automatically for unauthenticated users, there needs to be some sort of synchronous `fetchingUserID` state in the store to show when user info (if any) is still being loaded in the store. For now, when there is no `userID`, I disable the create button and show the message "You must be logged in to create a meetup." (The editing view is only shown when `userID === owner.id` so I haven't done any such checks for the edit controls.)

Reply:
If I were you, I'd just not implement the redirect by that point :)